config.json

```diff
@@ -819,7 +819,13 @@
       "core": false,
       "difficulty": 7,
       "slug": "binary-search-tree",
-      "topics": null,
+      "topics": [
+        "trees",
+        "sorting",
+        "searching",
+        "recursion",
+        "graphs"
```

Comment:
Great suggestions! The only one I'm not sure about is `recursion`. You could implement a binary search tree without recursion and as iteration is more suitable to Java, I would hesitate to add `recursion` as a topic.

I would add `generics` as well :)

Reply:
I agree about recursion, it is possible, but not the only way. Good idea about generics, added them as well.